src/core/vector/qgsvectorlayereditutils.cpp

```diff
@@ -521,8 +521,7 @@ Qgis::GeometryOperationResult QgsVectorLayerEditUtils::splitFeatures( const QgsC
           switch ( field.splitPolicy() )
           {
             case Qgis::FieldDomainSplitPolicy::DefaultValue:
-              // TODO!!!
-
+              //do nothing - default values ​​are determined
```

Comment:
This comment concerns not the duplicate policy but the split policy. I removed your "TODO" @nyalldawson because as far as I understand the default values are determined by doing "nothing".